src/amo/components/InstallButtonWrapper/index.js

```diff
@@ -92,6 +95,19 @@ export const InstallButtonWrapperBase = (props: InternalProps) => {
 
   const showDownloadLink = !isCompatible && installURL;
 
+  if (showButtonInsteadOfLink === false) {
+    return (
+      <div
+        className={makeClassName('InstallButtonWrapper', {
```

Comment:
This code (for `InstallButtonWrapper`) also appears twice in this file. Maybe it could appear just once and then conditional code inside it could decide whether to display the button or the link.